notebooks/preprocessing-workflow/tiling.ipynb

```diff
@@ -0,0 +1,229 @@
+{
```

Comment:
<p>Maybe just start with a sentence or two describing what the intent of the notebook is so novice users can decide if this notebook provides the example they are looking for? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WilhelmusLab/IceFloeTracker.jl/pull/434/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>What about a markdown cell at the top with a title and a sentence fulling that purpose?</p><p> </p><p>I think adding a README to the folder discussing what the notebooks are about might be useful also.</p>

Reply:
Yes, I think a markdown cell and an informative title would do the trick.